### PR TITLE
Update to v 10.42.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- version bumped up and sha256 updated.
 - build number reset to 0.
 - urls updated.

### CODE REVIEW

**Comment by akabanovs** [comment]
This update will require rebuilding a number of packages:

```
name	version	spec
git	2.34.1	pcre2 >=10.37,<10.38.0a0
git	2.32.0	pcre2 >=10.35,<10.36.0a0
libgdal	3.6.2	pcre2 >=10.37,<10.38.0a0
libgdal	3.6.0	pcre2 >=10.37,<10.38.0a0
libgit2	1.5.0	pcre2 >=10.37,<10.38.0a0
libgit2	1.0.1	pcre2 >=10.35,<10.36.0a0
libsolv	0.7.22	pcre2 >=10.37,<10.38.0a0
libsolv-static	0.7.22	pcre2 >=10.37,<10.38.0a0
nginx	1.21.6	pcre2 >=10.37,<10.38.0a0
r-base	4.2.0	pcre2 >=10.37,<10.38.0a0 
```

A separate epic will be created to updated these.

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {% set name = "pcre2" %}
-{% set version = "10.37" %}
+{% set version = "10.42" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/PhilipHazel/{{ name }}/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.tar.bz2
```
Comment (sumit0190) [inline recipe/meta.yaml]: You should update the URL here too.

**Review by sumit0190 — APPROVED** [review]
LGTM (after changing the download URL).

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Thanks for spotting.

**Pull request merged**